src/Files.App/MainWindow.xaml.cs

```diff
@@ -245,11 +257,15 @@ async Task PerformNavigationAsync(string payload, string selectItem = null)
 
 				if (rootFrame.Content is MainPage && MainPageViewModel.AppInstances.Any())
 				{
-					InteropHelpers.SwitchToThisWindow(WindowHandle, true);
+					// Bring to foreground (#14730)
+					AppLifecycleHelper.BringToForegroundEx(new(WindowHandle));
+
 					await NavigationHelpers.AddNewTabByParamAsync(typeof(PaneHolderPage), paneNavigationArgs);
 				}
 				else
+				{
 					rootFrame.Navigate(typeof(MainPage), paneNavigationArgs, new SuppressNavigationTransitionInfo());
+				}
```

Comment:
The backets can be removed

Reply:
I don't like this but I do that for now.

Reply:
Done